.gitignore

```diff
@@ -1,8 +1,13 @@
-/bin
+# IDE Specific
 /.idea
 *.iml
+
+# Holdovers from the ol' Ant system
```

Comment:
.classpath
.project
 and .externalToolBuilders are not from Ant. This is something so miniscule but it might be nice to move them up to IDE Specific